openpype/hosts/maya/plugins/create/create_rig.py

```diff
@@ -20,6 +20,13 @@ def create(self, subset_name, instance_data, pre_create_data):
         instance_node = instance.get("instance_node")
 
         self.log.info("Creating Rig instance set up ...")
+        # change name (_controls_set -> _rigs_SET)
         controls = cmds.sets(name=subset_name + "_controls_SET", empty=True)
+        # change name (_out_SET -> _geo_SET)
```

Comment:
These comments seem odd? Are these `# TODO`?
Also, this'd be a major backwards compatibility hazard ;)
Better leave these alone?